scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerHandshaker.java

```diff
@@ -717,16 +739,17 @@ private void negotiateCipherSuite(final ClientHello clientHello, final HelloExte
 
 		CertificateType supportedServerCertType = getSupportedServerCertificateType(clientHello);
```

Comment:
A bit out of topic, but why certtype are not a list ?

Reply:
Previous, because in the server_hello it is only possible to send one cert type, no list. Without sign_hash_algo and supported curves, this could be negotiated here. With them, it will get a little complicated. Therefore I postponed it to issue #1259

Reply:
My remarks was for both certtype. (ok to discuss about that later)